src/Components/ADT3DSceneBuilder/Internal/ConfirmDeleteDialog/ConfirmDeleteDialog.tsx

```diff
@@ -51,26 +54,23 @@ const ConfirmDeleteDialog: React.FC<Props> = ({
     return (
         <Dialog
             hidden={!isOpen}
-            onDismiss={() => {
-                onClose();
-            }}
+            onDismiss={onClose}
             dialogContentProps={confirmDeletionDialogProps}
             modalProps={confirmDeletionModalProps}
         >
+            {children}
             <DialogFooter>
                 <DefaultButton
-                    onClick={() => {
-                        onClose();
-                    }}
+                    onClick={onClose}
                     text={t('cancel')}
                     data-testid={'deleteDialog-cancel'}
                 />
                 <PrimaryButton
                     onClick={() => {
-                        onConfirmDeletion();
+                        onConfirm();
```

Comment:
should we wrap the event handler in a useCallback not to create it in every render?